src/WakuRlnV2.sol

```diff
@@ -59,24 +60,31 @@ contract WakuRlnV2 {
     /// @param index The index of the member in the set
     event MemberRegistered(uint256 idCommitment, uint32 userMessageLimit, uint32 index);
 
+    /// @notice the modifier to check if the idCommitment is valid
+    /// @param idCommitment The idCommitment of the member
     modifier onlyValidIdCommitment(uint256 idCommitment) {
         if (!isValidCommitment(idCommitment)) revert InvalidIdCommitment(idCommitment);
         _;
     }
 
-    modifier onlyValidUserMessageLimit(uint32 messageLimit) {
-        if (messageLimit > MAX_MESSAGE_LIMIT) revert InvalidUserMessageLimit(messageLimit);
-        if (messageLimit == 0) revert InvalidUserMessageLimit(messageLimit);
+    /// @notice the modifier to check if the userMessageLimit is valid
+    /// @param userMessageLimit The user message limit
+    modifier onlyValidUserMessageLimit(uint32 userMessageLimit) {
```

Comment:
we already test the modifier by testing for invalid user message limits and invalid id commitments as part of the registration proc.

Reply:
> and then someone modifies this modifier.
will it be used anywhere? it is imperative that the branch is tested. the code coverage result also yields if the modifiers have been used and which branches have not been tested. additionally, modifiers are just replaced inline to these functions by the compiler. I do not agree with this approach, we can't really call it "implicit 3rd party testing" since it is inlined with the functions its used in

Reply:
mm right. i can see also that testing modifiers in solidity is not a thing. the way to test them is literally, use them in a function and test the function xd. not sure i like that but that's another topic.

closing.